laborator/content/linking/demo/sections/sections.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+int a;
+int b = 10;
+char d;
+unsigned short array[3] = {10, 20, 30};
+char v[100];
+const int c = 10;
+static int stat = 40;
```

Comment:
Poate ar trebui sa scrii cate un comentariu langa fiecare in care precizezi daca e in `.data` `.rodata` sau `.bss`. Si as adauga si un exemplu de tipul `char *s = "Manele";`, unde pointerul si tringul sa fie stocate separat.

Reply:
O optiune ar fi sa adaugi si `char s2[] = "Manele";`, in antiteza cu `char *s = "Manele";`, dar deja asta tine si de limbaj, nu doar de sectiuni. Nu stiu daca merita.

Reply:
Nu pun si zonele ca de aici vin interbarile.